gatsby-node.js

```diff
@@ -19,6 +19,7 @@ exports.createPages = ({ graphql, actions }) => {
               filter: {
                 fields: {
                   slug: {
+                    nin: ["", "nodejs-community", "homepage", "working-groups"]
```

Comment:
You should be adding working groups to the existing array not creating a new one. I'm not sure how this was building for you locally 🤔